samples/ChatSample/ChatSample.csproj

```diff
@@ -3,13 +3,17 @@
   <Import Project="..\..\build\dependencies.props" />
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFrameworks>netcoreapp2.0;net461</TargetFrameworks>
     <UserSecretsId>aspnet-ChatSample-f11cf018-e0a8-49fa-b749-4c0eb5c9150b</UserSecretsId>
     <PackageTargetFallback>$(PackageTargetFallback);portable-net45+win8</PackageTargetFallback>
```

Comment:
I still don't know why we need this fallback (what happens if it were removed). It also looks different from the fallback you added here: https://github.com/aspnet/SignalR/pull/486/files/46a28efc05b931c8b0389fa7b1e7041a3e6bd8c1..574e53b6514e34f40f4d04c38b8b72818afc8446#diff-502c00f36596d80fb0e0159c90a0b43dR7